docker-compose-tahoe.yml

```diff
@@ -7,9 +7,6 @@ services:
   chrome:
     image: hello-world
     command: /hello
-  elasticsearch:
```

Comment:
how does removing this enable search on devstack?

Reply:
Good question: `elasticsearch: image: hello-world` was effectively turning off `elasticsearch` by making it doing nothing.

This is an old hack that is confusing, I've now cleaned it up.